### DISCUSSION TITLE
Are the generations from Tortoise free to use for YouTube videos?

### DISCUSSION THREAD

**Discussion by Simplegram** [neonbjb/tortoise-tts#683] [General]
Just wondering if I can use the voices commercially (YouTube, school project, etc)

**Comment by fakerybakery** [comment]
Yes, please refer to the license for details. The Apache 2.0 license is a permissive license that allows for commercial use, with a few restrictions regarding redistribution of the source code. IANAL

**Reply by Simplegram** [reply]
Oh yeah, forgot to check the license. Thank you!
